dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/clients/common/OidcAuthorizationRequest.java

```diff
@@ -253,10 +262,10 @@ void addPkceParameters(String codeChallengeMethod, String state, AuthorizationRe
     void addImplicitParameters(AuthorizationRequestParameters authzParameters) throws UnsupportedEncodingException {
         authzParameters.addParameter("response_mode", "form_post");
         // add resource
-        String resources = getResourcesParameter();
-        if (resources != null) {
-            authzParameters.addParameter("resource", resources);
-        }
+        //        String resources = getResourcesParameter();
+        //        if (resources != null) {
+        //            authzParameters.addParameter("resource", resources);
+        //        }
```

Comment:
If the code isn't needed anymore, just delete it instead of commenting it out.

Reply:
@ayoho thank you for the comments. I will do the cleanup as suggested.